package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "nodule-openapi",
+    "version": "0.1.0",
+    "description": "Opinionated OpenAPI (Swagger) Client",
+    "main": "src/index.js",
+    "repository": "git@github.com:globality-corp/nodule-openapi.git",
+    "author": "Globality Engineering",
+    "license": "MIT",
+    "scripts": {
+        "build": "babel src --out-dir dist --ignore '**/__tests__/*,**/__mocks__/*'",
```

Comment:
out dir should be `lib/` for a project like that. It's a common pattern for node library projects

Reply:
So just `s/dist/lib` ?

Reply:
No just lib...
Usually, an index.js is present in the /lib folder as the main package's entry point. This file usually exports the "public API" of the package